webpack/dev.config.js

```diff
@@ -10,7 +10,21 @@ module.exports = (params) => {
     conf.resolve.modules.push('dev');
     conf.resolve.symlinks = false;
     conf.module.exprContextCritical = false;
-    conf.module.loaders.unshift({test: /\.jsx?$/, exclude: /(node_modules(\\|\/)(?!(impl|ut|.*dfsp)\-).)/, loaders: ['react-hot', 'babel?presets[]=es2015&presets[]=stage-0&presets[]=react&cacheDirectory=true']});
+    conf.module.rules.unshift({
+        test: /\.jsx?$/,
+        exclude: /(node_modules(\\|\/)(?!(impl|ut|.*dfsp)-).)/,
+        use: [{
+            loader: 'react-hot-loader/webpack'
+        }, {
+            loader: 'babel-loader',
+            options: {
+                presets: ['es2015', 'stage-0', 'react'],
```

Comment:
is it tested, because there was webpack loader config changes for babel, i will not accept presets as options defined like this, it should be passed as query props